.storybook/webpack.config.js

```diff
@@ -19,7 +19,7 @@ const {
   ADMIN_URL,
   APP_URL,
   CI,
-  COLLECTION_HUBS,
+  COLLECTION_HUB_ENTRYPOINTS,
```

Comment:
Echoing my comment [from the Force PR](https://github.com/artsy/force/pull/4582/files#r327790289) - 

- I think we need two variables, one for the home page and one for collect.
- I also think it would be useful to include "EXPERIMENT" in the name(s), to tip us off that it is an experiment.